rotkehlchen/tests/unit/decoders/test_arbitrum_one_bridge.py

```diff
@@ -388,3 +388,43 @@ def test_receive_erc20_on_ethereum(database, ethereum_inquirer, ethereum_account
             address=string_to_evm_address('0x6A23F4940BD5BA117Da261f98aae51A8BFfa210A'),
         ),
     ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+def test_receive_erc20_on_ethereum_old_bridge(database, ethereum_inquirer, ethereum_accounts):
+    evmhash = deserialize_evm_tx_hash('0xdd71f6b50a24b2f6704819579cb4c0d27cf3d56bd3ba03fe8a7a9f9dc56eea52')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    user_address = ethereum_accounts[0]
```

Comment:
can you do what we do in other tests moving the constants in the test to variables to avoid repeating them? timestamp, user, gas amount, withdrawn amount...